src/main/resources/application-local.properties

```diff
@@ -15,8 +15,14 @@ keycloak.clientId=my-app
 keycloak.clientSecret=xq1kKt43x64JMS1krJhsmo6sqG3rYkfH
 keycloak.tokenUri=http://localhost:8085/realms/app/protocol/openid-connect/token
 keycloak.logoutUri=http://localhost:8085/realms/app/protocol/openid-connect/logout
+quarkus.transaction-manager.enable-recovery=true
+quarkus.http.port=8085
```

Comment:
What are these properties about? We have spring boot up. What are Quarkus properties doing in our project? 
You are again using properties you don't understand the meaning of(((((